### ISSUE TITLE
app does not work

### ISSUE THREAD

**Issue by aberlin** [lsubel/amam-cordova#1]
When i start the app I get only a white screen whithout an obvious error message.
I used F-droid to get the app ( v 1.3.2). I use Android 4.2.2

**Comment by lsubel** [comment]
Confirmed. Appearantly, one of the questionnaire was ill-formed. An updated version 1.3.3 was released and will be compiled by F-Droid in the near future.
Thank you for the hint! :-)

**Comment by mwild1** [comment]
Hey. I'm seeing the same problem, but fdroid says I have 1.3.3 installed... :)

**Comment by lsubel** [comment]
Which Android version are you using? If you want to help me debugging the problem, you could send me the logcat output when you start the application.

**Comment by est31** [comment]
1.3.3 works for me.

**Comment by aberlin** [comment]
Not for me

I hope this helps: (catlog) https://gist.github.com/aberlin/0a1d9dff7522b231fe3bc91ce3541fa5

I hope it is not a dumb error on my side.

Its short because I dont know what is Important.
Please ask if you need something specific.

**Comment by lsubel** [comment]
Appearantly the WebView had in some cases no access to the requestAnimationFrame function. I have added a fallback function.

**Comment by hydroid7** [comment]
Same issue with the latest Version on Android 4.4.2

**Comment by lsubel** [comment]
A new version is now available. Is the error still present?

**Comment by mwild1** [comment]
Sadly yes. 1.3.4 on Android 4.4.2. This is from my logcat: https://gist.github.com/mwild1/82cf644b30b3fd73f253aa0e594d2f97

**Comment by aberlin** [comment]
Also for android 4.2.2
I have no Gapps installed. But this shouldn't matter.

**Comment by hydroid7** [comment]
Yeah me too. 

Link to Screenshot

https://www.anony.ws/i/2016/04/18/Screenshot_2016-04-18-22-19-04.png

**Comment by lsubel** [comment]
I guess more details are necessary to find the root problem.
I have added the debug apks built by Cordova to the repository. You can find them [here](https://github.com/lsubel/amam-cordova/tree/master/platforms/android/build/outputs/apk). Could you guys install the android-debug.apk on your device and give me the logcat infos? :-)

**Comment by aberlin** [comment]
Hope this helps: https://gist.github.com/aberlin/5a7b01e459961c88120207f2cacfa04c

**Comment by lsubel** [comment]
Unfortunately not. But additional logcat entries are welcomed.
Please check whether you can access and navigate the [web](https://lsubel.github.io/amam/) version.

**Comment by aberlin** [comment]
Good idea. It works with Firefox but not with the internal browser.

**Comment by lsubel** [comment]
I finally found the reason for the error.
Apparently, the WebView element up to Android 4.3 does not support methods used by the application, including the requestAnimationFrame method ([link](http://caniuse.com/requestanimationframe)) and navigator.language for user language detection. Therefore, I have added some workarounds.
The updated application was tested on an Android 4.2.2 emulator and seems to start. However, there were some back fragments present. Since I cannot they whether they are the result of the emulator or not, I'd like to ask @aberlin to test the updated [debug apk](https://github.com/lsubel/amam-cordova/tree/master/platforms/android/build/outputs/apk).
